packages/@o3r/create/src/index.ts

```diff
@@ -126,7 +126,7 @@ const createNgProject = () => {
   }
 };
 
-const addOtterCore = (relativeDirectory = '.') => {
+const addOtterFramework = (relativeDirectory = '.') => {
   const cwd = resolve(process.cwd(), relativeDirectory);
   const { error } = spawnSync(process.execPath, [binPath, 'add', `@o3r/core@${version || 'latest'}`, ...args.filter((arg) => arg.startsWith('-'))], {
```

Comment:
I think here we have to add the `@o3r/workspace` package to setup the root rules

Reply:
Nop, it is actually added by `ng add @o3r/core` if installed at root level and if `@o3r/workspace` is not detected